HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -400,7 +400,15 @@
         'keep *_hltScoutingPFPacker_*_*',
         'keep *_hltScoutingPrimaryVertexPacker_*_*',
         'keep *_hltScoutingTrackPacker_*_*',
-        'keep edmTriggerResults_*_*_*'
+        'keep edmTriggerResults_*_*_*',
+        'keep *_hltL2MuonSeedsFromL1TkMuon_*_*',
```

Comment:
please do NOT change this file directly as it is used also for the Run3 HLT menu and its content synchronized with confDB. Change instead `Configuration/EventContent/python/EventContent_cff.py` as done e.g. in https://github.com/cms-sw/cmssw/pull/46823